libcontainer/factory_linux.go

```diff
@@ -256,11 +256,13 @@ func (l *LinuxFactory) StartInitialization() (err error) {
 		if _, ok := i.(*linuxStandardInit); ok {
```

Comment:
I just tried adding an `fmt.Fprintln(os.Stderr, "some text")` here to see what happens. It looks like there's still a lot of console tomfoolery going on -- because we set `cmd.Process.Std{in,out,err} = nil` before running `runc init`. So the container bootstrap doesn't write to the *actual* stdio (it writes to `/dev/null`). The reason I went with `panic`s here is because of that issue.